CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to CPS
+
+Thank you for your interest in contributing to the Common Package Specification (CPS) project!
+
+We welcome contributions of all kinds, including bug reports, feature requests, documentation improvements, and code changes.
+
+## How to Contribute
+
+### 1. Reporting Issues
+
+- Please use [GitHub Issues](https://github.com/cps-org/cps/issues) to report bugs or suggest enhancements.
+- When reporting a bug, include as much detail as possible: steps to reproduce, expected behavior, actual behavior, and relevant logs or code snippets.
+
+### 2. Submitting Pull Requests
+
+- Fork the repository and create your branch from `main`.
+- Make your changes with clear, descriptive commit messages.
+- If your change affects the specification or documentation, please update the relevant files.
+- Ensure your code passes all tests and style checks.
+- Open a pull request and describe your changes and motivation.
```

Comment:
I don't think we need to maintain content giving users a tutorial on how to create pull requests. If we want a section on pull requests, we should just say that they are welcome. But we should mention that substantial changes should start as a discussion, for instance in an issue on this repository.